pkg/cloudprovider/gce/gce.go

```diff
@@ -309,6 +309,7 @@ func translateAffinityType(affinityType api.AffinityType) GCEAffinityType {
 // CreateTCPLoadBalancer is an implementation of TCPLoadBalancer.CreateTCPLoadBalancer.
 // TODO(a-robinson): Don't just ignore specified IP addresses. Check if they're
 // owned by the project and available to be used, and use them if they are.
+// TODO: Verify this is idempotent
```

Comment:
Sorry, I very well may be missing something about it - what do you see in it that might not be idempotent?